### ISSUE TITLE
placeholders %height% %width% %length% are not showing in mm (unit set on view > unit like inch, m, mm ...)

### ISSUE THREAD

**Issue by manuelfink** [jgraph/drawio#4362]
### Preflight Checklist

* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**Describe the bug**
The placeholders %height% %width% %length% within field texts are not shown in the localized unit of measurement like mm, cm, m, inch. Placeholders as of april 2024 only show in pt.

**To Reproduce**
1. set unit of measurement (uom) to mm under view > unit > mm (cm is also missing as uom)
2. create any rectangle box and size it 50mm x 50m
3. right click, edit data, activate placeholders and apply
4. edit the content of the box and ender %height%, %width% (and %length% for a line).

**Expected behavior**
placeholders like %height%, %width%, %length% should show the size in the unit of measurement like mm.

**Screenshots**

<img width="519" alt="Bildschirmfoto 2024-04-28 um 15 34 17" src="https://github.com/jgraph/drawio/assets/3414822/24b92766-5f8a-4428-9181-e6fcb9b2e496">

**draw.io version (In the Help->About menu of the draw.io editor):**

- draw.io version v24.2.5 (latest)

**Desktop (please complete the following information):**
 - OS: iOS
 - Browser Chrome
 - Browser Version 123.0.6312.107 (Offizieller Build) (x86_64)

**I tested the problem in incognito/private mode with all browser extensions switched off, write "yes" below:**
- yes

**Additional context**

This issue relates to following open issues, but is different in context, as it refers to a different area of the application

* https://github.com/jgraph/drawio/issues/713
* https://github.com/jgraph/drawio/issues/3327

**Comment by greatvovan** [comment]
Agree, dimensions placeholders are currently useless as they display only in pt.

**Comment by gwlewis** [comment]
Is there currently any way to resolve placeholders units to match the document?

**Comment by DeadRabbits307** [comment]
any progress on this topic? 
Version 25.0.3 in the browser still shows the described behavior.

**Comment by alvangee** [comment]
I see the issue is closed but was it resolved?

Online version 24.7.17 still shows length of connector only in pt.
Desktop 26.0.9 too.

**Comment by alderg** [comment]
<img width="103" alt="Image" src="https://github.com/user-attachments/assets/fcbc2a2a-a213-4ea3-b8ff-a96870a86abd" />

**Comment by alvangee** [comment]
> <img alt="Image" width="103" src="https://private-user-images.githubusercontent.com/921200/415157228-fcbc2a2a-a213-4ea3-b8ff-a96870a86abd.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDAwNTE0MzgsIm5iZiI6MTc0MDA1MTEzOCwicGF0aCI6Ii85MjEyMDAvNDE1MTU3MjI4LWZjYmMyYTJhLWEyMTMtNGVhMy1iOGZmLWE5Njg3MGE4NmFiZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMjIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDIyMFQxMTMyMThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hOGIwY2I5NGQ4Zjk5ZjJhMzdmNTUyZjRhYzZiMWJiZmI5ZWZhMGQwMjI0ZWFjZGZlNWNjZGE5YmQ5OGU3MGI0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.qpUcasy4zUE7vICrYPhYACxdmkb9aXa-v5wZEQH7PHk">

Thank you! 
I had trouble accessing drawio documentation hence the unnecessary question.
Now that it finally opened (only under VPN) I see that format %length:mm% also works and can confirm this.
Sorry for bothering.
